bitnami/postgresql/UPGRADING.md

```diff
@@ -0,0 +1,329 @@
+# Upgrading Notes
```

Comment:
Could you confirm that `helm package` includes this file in the tarball? Could you confirm what happens in Marketplaces?

Reply:
> Could you confirm that helm package includes this file in the tarball?

Yes, it includes it.

> Could you confirm what happens in Marketplaces?

They'll be referencing to a file that's not browsable. I'll move this section to [charts-docs](https://github.com/bitnami/charts-docs)

Reply:
See https://github.com/bitnami/charts-docs/pull/93